agent/agentbootstrap/bootstrap.go

```diff
@@ -231,6 +231,13 @@ func (b *AgentBootstrap) Initialize(ctx stdcontext.Context) (_ *state.Controller
 	// and a function to insert it into the database.
 	adminUserUUID, addAdminUser := userbootstrap.AddUserWithPassword(b.adminUser.Name(), auth.NewPassword(info.Password))
 
+	// Add initial metrics user to the database.
+	metricsPassword, err := password.RandomPassword()
```

Comment:
I assume we don't use this as part of bootstrap, so I'd have this in the bootstrap worker.

Reply:
Sure I can change that.